keyberon/src/action.rs

```diff
@@ -84,7 +84,7 @@ pub enum HoldTapConfig<'a> {
     /// value will cause a fallback to the timeout-based approach. If the
     /// timeout is not triggered, the next tick will call the custom handler
     /// again.
-    Custom(&'a (dyn Fn(QueuedIter) -> Option<WaitingAction> + Send + Sync)),
+    Custom(&'a (dyn Fn(QueuedIter) -> (Option<WaitingAction>, bool) + Send + Sync)),
```

Comment:
Please add to documentation about what the `bool` return value is meant for.

Reply:
should be correct now